charts/client/chart.scss

```diff
@@ -14,15 +14,13 @@ $zindex-Tooltip: 20;
 $zindex-EntitySelector: 30;
 $zindex-IndicatorDropdown: 50;
 
-figure[data-grapher-src]:empty,
-.loading-chart {
+figure[data-grapher-src]:empty {
```

Comment:
I wasn't really sure in which case `figure[data-grapher-src]:empty` applies, if any.
If it isn't being used we can remove these 30 lines here, but I didn't wanna break anything just yet.

Reply:
It applies while JavaScript is loading on a `/grapher/` page.

Reply:
Ah, makes sense.
I don't like the duplication we have here, but can't really come up with a nicer solution either (except for changing the HTML of the `/grapher` pages, but that's not nice either).

Maybe a CSS mixin for the loading indicator?

Reply:
Okay, I've now gone with including them server-side.
The LoadingIndicator is then replaced with the interactive chart once React renders it.